central/processlisteningonport/datastore/datastore_impl.go

```diff
@@ -61,13 +61,20 @@ func (ds *datastoreImpl) AddProcessListeningOnPort(
 	} else if !ok {
 		return sac.ErrResourceAccessDenied
 	}
+	for _, plop := range portProcesses {
+		log.Infof("portProcess: %+v", plop)
+	}
 
 	normalizedPLOPs, completedInBatch := normalizePLOPs(portProcesses)
+	allNormalizedPLOPs := append(normalizedPLOPs, completedInBatch...)
```

Comment:
See https://github.com/stackrox/stackrox/pull/5204 to understand why this and the following change was made.

Reply:
#5204 is about fixing a lookup for process indicators with empty query. Since there is no lookup, why is it needed?

Reply:
The query returns an error if it is empty. Without this change we get empty queries in the unit tests.